### PR TITLE
Fixes 4852: close pgpool after integration tests

### PR DESCRIPTION
**Pull request by jlsherrill**
## Summary
run the integration tests
```
make test-integration
```

while its running check db connections:
```
make db-cli-connect


select * from
(select count(*) used from pg_stat_activity) q1,
(select setting::int res_for_super from pg_settings where name=$$superuser_reserved_connections$$) q2,
(select setting::int max_conn from pg_settings where name=$$max_connections$$) q3;
```

Prior to this is would grow as the tests ran to approach the 100 default limit we have through our docker compose

### CODE REVIEW

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/HMS-TODO

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/HMS-4852

File: test/integration/pgtest_test.go.bak
Comment (rverdile) [inline test/integration/pgtest_test.go.bak:1]: was this left in accidentally?

File: pkg/handler/api.go [hunk 1]
```diff
@@ -60,7 +61,7 @@ func RegisterRoutes(engine *echo.Echo) {
 		pgqueue queue.PgQueue
 	)
 	paths := []string{api.FullRootPath(), api.MajorRootPath()}
-	pgqueue, err = queue.NewPgQueue(db.GetUrl())
+	pgqueue, err = queue.NewPgQueue(context.TODO(), db.GetUrl())
```
Comment (rverdile) [inline pkg/handler/api.go:64]: should this be changed to context.Background()? unless you anticipate the source of this context changing

File: test/integration/pgtest_test.go.bak
Comment (jlsherrill) [reply test/integration/pgtest_test.go.bak:1]: oops yes!

File: pkg/handler/api.go (hunk 1 above)
Comment (jlsherrill) [reply pkg/handler/api.go]: wasn't too bad, updated

File: test/integration/pgtest_test.go.bak
Comment (rverdile) [reply test/integration/pgtest_test.go.bak:1]: if you meant to remove it, it's still there

File: pkg/handler/api.go (hunk 1 above)
Comment (jlsherrill) [reply pkg/handler/api.go]: now it should be gone :)

**Review by rverdile — APPROVED** [review]
looks good!

**Pull request merged**